upstream svn31547 fix for dvips
---
 texk/dvipsk/ChangeLog |    5 +++++
 texk/dvipsk/writet1.c |    4 +++-
 2 files changed, 8 insertions(+), 1 deletion(-)

 2013-08-05  Akira Kakuto  <kakuto@fuk.kindai.ac.jp>
 
 	* bbox.c: Fix a bug in bbdopage() to obtain correct BoundingBox in
--- texk/dvipsk/writet1.c
+++ texk/dvipsk/writet1.c
@@ -1167,7 +1167,9 @@
     t1_getline();
     while (!t1_prefix("/Encoding")) {
         t1_scan_param();
-        t1_putline();
+        if (!(t1_prefix("/UniqueID") &&
+            !strncmp(t1_line_array + strlen(t1_line_array) -4, "def", 3)))
+            t1_putline();
         t1_getline();
     }
     if (is_reencoded(fm_cur))
